docs/_quickstart/build_trigger.md

```diff
@@ -36,70 +36,81 @@ Add those details to your GitHub new issue trigger, then save and continue.
+![Zapier Visual Builder input field details](https://cdn.zappy.app/f6826e50873c92618df64a8b797e49e2.png)
 
 _Need to gather data from users for your trigger? Add input form fields to your integration to do so._
 
-We want to filter GitHub issues by repository and issue type, which means this trigger needs input fields.
+We want to use the List repository issues endpoint [detailed here](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues) so we’ll need input fields for this trigger as the endpoint is `/repos/{owner}/{repo}/issues`
 
-Add a _User Input Field_ in the _Input Designer_ tab first, where users can add the repository they want to watch for issues. Enter a label, key, and help text for the repo field—and set it as required.
+Add a _Repo Input Field_ in the _Input Designer_ tab first, where users can add the repository they want to watch for issues. Enter a label, key, and help text for the repo field—and set it as required.
 
-To make this example integration simple, use a `string` field type where the user can type the repo name into Zapier instead (then for a challenge later, you could try making a dynamic trigger that has Zapier fetch the users' repositories and show them in a list).
+To make this example integration simple, use a `string` field type where the user can type the repo name into Zapier instead (then for a challenge later, you could try making a dynamic trigger that has Zapier fetch the user's repositories and show them in a dropdown list).
 
-> Learn more about Input forms in our detailed [input designer docs](https://platform.zapier.com/docs/input-designer).
+> Learn more about Input Field forms and dynamic dropdowns in our detailed [input designer docs](https://platform.zapier.com/docs/input-designer).
+
+Your GitHub integration needs an additional input field: an _Owner_ field for users to enter the username of the owner of that repository.
 
 ![Zapier Visual Builder dropdown](https://cdn.zapier.com/storage/photos/992bbec7742b65a3b48df6c981e58610.png)
 
 _You can also build a dropdown menu with dynamic or static options_
 
-Your GitHub integration needs two additional input fields: a _Username_ field for users to enter the username of the owner of that repository, and a _Filter_ field to choose which type of issues to trigger the Zap. Repeat the steps to add the username field then the Filter field, and with the Filter check the _Dropdown_ box and the _Static_ option. Enter the following in the code box to build your menu, then save your field:
+Since _State_ is an available parameter for this endpoint, let’s add that too as it will be useful to be able to choose which type of issues should trigger the Zap. Repeat the steps to add the _Owner_ field then the _State_ field, and with the _State_, add a default value of _open_ and check the _Dropdown_ box and the _Static_ option. Enter the following in the code box to build your menu, then save your field:
```

Comment:
```suggestion
Since _State_ is an available parameter for this endpoint, let’s add that too as it will be useful to be able to choose which type of issues should trigger the Zap. Repeat the steps to add the _Owner_ field then the _State_ field. For the _State_ field, add a default value of _open_ and check the _Dropdown_ box and the _Static_ option. Enter the following in the code box to build your menu, then save your field:
```